From a4acf87dfb8e3f7539c00d81c640132e2e9893f2 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 26 Mar 2023 11:51:58 +0300
Subject: [PATCH 48/50] arm64: dts: lx2160a-clearfog-itx: add qsfp connector

Describe the qsfp connector with i2c bus and module detect pin.
Since QSFP is not supported (yet), leave the node disconnected from net
devices for now.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 .../arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi b/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
index b201512c7382..86036bd39b6a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
@@ -31,6 +31,13 @@ key {
 		};
 	};
 
+	qsfp0: qsfp-0 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c2>;
+		mod-def0-gpio = <&gpio2 5 GPIO_ACTIVE_LOW>; // GPO2 IRQ05 GPIO3_DAT05
+		maximum-power-milliwatt = <8000>;
+	};
+
 	sfp0: sfp-0 {
 		compatible = "sff,sfp";
 		i2c-bus = <&sfp0_i2c>;
-- 
2.35.3

